src/Main.js

```diff
@@ -242,10 +242,19 @@ CM.DelayInit = function() {
 	if (Game.prefs.popups) Game.Popup('Cookie Monster version ' + CM.VersionMajor + '.' + CM.VersionMinor + ' loaded!');
 	else Game.Notify('Cookie Monster version ' + CM.VersionMajor + '.' + CM.VersionMinor + ' loaded!', '', '', 1, 1);
 
+	CM.RegisterHooks();
```

Comment:
Really nitpicky, but could you change this to be CM.Main.RegisterHooks()?

Reply:
No worries, I'll do it right away!

Reply:
Done!